build-system/tasks/coverage-map/index.js

```diff
@@ -66,8 +66,9 @@ async function collectCoverage() {
 
 /**
  * Source: https://github.com/chenxiaochun/blog/issues/38s
- *
- * @param {puppeteer.Page} page
+ * TODO(28387) change the first parameter to puppeteer.Page once puppeteer is consistently
```

Comment:
nit: linkify
```suggestion
 * TODO(#28387) change the first parameter to puppeteer.Page once puppeteer is consistently
```

Same comment applies to the rest of the TODOs in this PR.

Reply:
nit: Unresolving because there are a few more instances of `TODO(XXXXX)` that can be linkified with a `#`.

Reply:
Resolved for real this time